proxy/http/HttpSM.cc

```diff
@@ -24,6 +24,7 @@
 
 #include "../ProxyClientTransaction.h"
 #include "HttpSM.h"
+#include "HttpTransact.h"
```

Comment:
This include appeared to be missing earlier. Since we are adding a helper in this header, I am adding it here.